extensions/amp-instagram/1.0/component.js

```diff
@@ -15,34 +15,99 @@
  */
 
 import * as Preact from '../../../src/preact';
```

Comment:
Why are we porting over amp-instagram instead of just bento components? Can we separate out this diff? If a revert is needed, would hate to have to undo Bento migration as well